src/overlays/actors/ovl_En_Viewer/z_en_viewer.h

```diff
@@ -3,14 +3,23 @@
 
 #include "global.h"
 
+#define ENVIEWER_GET_3(thisx) ((thisx)->params & 3)
+#define ENVIEWER_GET_9(thisx) ((thisx)->params >> 9)
+#define ENVIEWER_GET_7F(thisx) (((thisx)->params >> 2) & 0x7F)
```

Comment:
This is also not 7F, it's `0x7F << 2`, whatever that is.

Reply:
please write the example macro for me

Reply:
Again just referring to the name, you can put `0x7F << 2` in a calculator/google and it will tell which bits this is referring to.